src/cargo/ops/cargo_compile.rs

```diff
@@ -796,6 +813,21 @@ impl CompileFilter {
         }
     }
 
+    pub fn is_bare_install(&self) -> bool {
```

Comment:
I'm a little unclear, why doesn't this just check for the `warn_unmatched` value?